battery/df_manip.py

```diff
@@ -371,34 +353,16 @@ def extract_profiles(file='profiles.out', path=None):
     Parameters
     ----------
     file : str
-        main output file (most likely "Dualfoil5.out") generated by Dualfoil5.2
+        main output file (most likely "profiles.out") generated by Dualfoil
     path : str
         path to get to file, if not in current directory
 
     Returns
     -------
-    time : list of floats
-        the time in seconds each profile is taken
-    distance : list of floats
-        the distance accross the cell in microns
-    elec_conc : list of list of floats
-        the concentration of electrolyte in mol/cubic meters
-    sol_surf : list of list of floats
-        Potential in the electrolyte, neg. or pos. depending on location in cell
-    liq_potential : list of list of floats
-        the liquid potential in volts
-    sol_potential : list of list of floats
-        the solid potential in volts
-    liq_cur : list of list of floats
-        the liquid current density in amperes/square meters
-    jmain : list of list of floats
-        main liquid current density in amperes/square meters
-    j_side1 : list of list of floats
-        first side reaction liquid current density in amperes/square meters
-    j_side2 : list of list of floats
-        second side reaction liquid current density in amperes/square meters
-    j_side3 : list of list of floats
-        third side reaction liquid current density in amperes/square meters
+    dict
+        Contains lists for each type of input
+        *Note: 'time' is a list of floats; every other key is two-dimmensional
+               ([number of timesteps] by [number of nodes across cell])
```

Comment:
same thing. By reading the documentation, I don't know what the dictionary contains.